setup.py

```diff
@@ -38,7 +38,7 @@ def load_about():
     long_description=load_readme(),
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
-    python_requires=">=3.7",
+    python_requires=">=3.8",
     install_requires=["tutor", "packaging"],
```

Comment:
Should we start adding a constraint to the Tutor version? As [other plugins](https://github.com/overhangio/tutor-notes/blob/v16.0.2/setup.py#L35) do